artemis-server/src/main/java/org/apache/activemq/artemis/core/settings/impl/AddressSettings.java

```diff
@@ -127,6 +127,8 @@
 
    public static final boolean DEFAULT_ENABLE_METRICS = true;
 
+   public static final int MANAGEMENT_MESSAGE_ATTRIBUTE_SIZE_LIMIT = 256;
```

Comment:
Surely want existing behaviour of unlimited so users dont get shocks.

Reply:
nope. If there is a 1k property, we don't want that 1k on the browser or on the jmx command line as ascii. The fact that we currently display the full values is a bug imho.

Reply:
i cannot resolve my comment but treat this as resolved.